gatsby-theme-landing-page/src/components/copy.module.css

```diff
@@ -0,0 +1,52 @@
+.root {
+  padding-top: var(--space-5);
+  padding-bottom: var(--space-5);
+}
+
+p,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
```

Comment:
Does this stay scoped to the component? My CSS Modules is a little rusty, but we want to make sure theme styles don't leak into the consuming site

Reply:
it doesn't; updating!